web/src/components/Schedule/Schedule.jsx

```diff
@@ -47,6 +47,14 @@ export default function Schedule() {
     getAllEvents();
   }
 
+  async function deleteEvent(id) {
```

Comment:
I would avoid including this file in the pull request as the frontend in main is entirely different now. I personally would make the backend-frontend connection in a later PR.

Reply:
makes sense, just removed the front-end in the latest commit!!